### PR TITLE
missing plugin slug in tests/bootstrap.php

### PR DESCRIPTION
**Pull request by miya0001**
When I ran `wp scaffold plugin hello-world` then `tests/bootstrap.php` was like following.

```
<?php

$_tests_dir = getenv( 'WP_TESTS_DIR' );
if ( ! $_tests_dir ) {
	$_tests_dir = '/tmp/wordpress-tests-lib';
}

require_once $_tests_dir . '/includes/functions.php';

function _manually_load_plugin() {
	require dirname( __FILE__ ) . '/../.php'; // plugin slug missing
}
tests_add_filter( 'muplugins_loaded', '_manually_load_plugin' );

require $_tests_dir . '/includes/bootstrap.php';
```

### CODE REVIEW

File: features/scaffold.feature [hunk 1]
```diff
@@ -151,6 +151,10 @@ Feature: WordPress code scaffolding
       bootstrap.php
       test-sample.php
       """
+    And the {PLUGIN_DIR}/hello-world/tests/bootstrap.php file should contain:
+      """
+      hello-world.php
```
Comment (danielbachhuber) [inline features/scaffold.feature]: Can you have this test for the entire line?

**Comment by danielbachhuber** [comment]
Good catch :+1: Might be worth a patch release for this. Just one comment.

**Comment by miya0001** [comment]
Thanks. :smile: 

> [Can you have this test for the entire line?](https://github.com/wp-cli/wp-cli/pull/1828#discussion-diff-29996712)
You mean that you want it to be like this?

**Comment by danielbachhuber** [comment]
> You mean that you want it to be like this?

Yes, that's great.

There's another bug, however. If you specify just `--dir=path/to/directory`, the `$plugin_slug` doesn't appear to be set. Can you fix in this PR too? Here's the relevant tests: https://github.com/wp-cli/wp-cli/blob/master/features/scaffold.feature#L269

**Comment by miya0001** [comment]
Oh I've merged #1827 by mistake.
But it is problem with same command.
I want to close #1827. Is it OK?

File: templates/bootstrap.mustache [hunk 2]
```diff
@@ -8,9 +8,8 @@ if ( ! $_tests_dir ) {
 require_once $_tests_dir . '/includes/functions.php';
 
 function _manually_load_plugin() {
-	require dirname( __FILE__ ) . '/../.php';
+	require dirname( __FILE__ ) . '/../{{plugin_slug}}.php';
```
Comment (szepeviktor) [inline templates/bootstrap.mustache]: Could you go `dirname()`?
```php
require dirname( dirname( __FILE__ ) ) . '/{{plugin_slug}}.php';
```

Comment (miya0001) [inline templates/bootstrap.mustache]: Oh... Nice :+1:  I like it!

**Comment by danielbachhuber** [comment]
> But it is problem with same command.
> I want to close #1827. Is it OK?

Yep, this is fine.

File: templates/install-wp-tests.sh [hunk 3]
```diff
@@ -62,12 +62,12 @@ install_test_suite() {
 	cd $WP_TESTS_DIR
 
 	if [ ! -f wp-tests-config.php ]; then
-		download https://develop.svn.wordpress.org/trunk/wp-tests-config-sample.php wp-tests-config.php
-		sed $ioption "s:dirname( __FILE__ ) . '/src/':'$WP_CORE_DIR':" wp-tests-config.php
-		sed $ioption "s/youremptytestdbnamehere/$DB_NAME/" wp-tests-config.php
-		sed $ioption "s/yourusernamehere/$DB_USER/" wp-tests-config.php
-		sed $ioption "s/yourpasswordhere/$DB_PASS/" wp-tests-config.php
-		sed $ioption "s|localhost|${DB_HOST}|" wp-tests-config.php
+		download https://develop.svn.wordpress.org/trunk/wp-tests-config-sample.php ${WP_TESTS_DIR}/../wp-tests-config.php
```
Comment (szepeviktor) [inline templates/install-wp-tests.sh]: `$(dirname ${WP_TESTS_DIR})/wp-tests-config.php`

Comment (miya0001) [inline templates/install-wp-tests.sh]: :wink:

Comment (szepeviktor) [inline templates/install-wp-tests.sh]: Thank you!

File: templates/install-wp-tests.sh [hunk 4]
```diff
@@ -11,7 +11,7 @@ DB_PASS=$3
 DB_HOST=${4-localhost}
 WP_VERSION=${5-latest}
 
-WP_TESTS_DIR=${WP_TESTS_DIR-/tmp/wordpress-tests-lib}
+WP_TESTS_DIR=${WP_TESTS_DIR-/tmp/wordpress-tests-lib/includes}
```
Comment (danielbachhuber) [inline templates/install-wp-tests.sh]: @miya0001 Why did you change this originally?

Comment (miya0001) [inline templates/install-wp-tests.sh]: I thought, `WP_UnitTestCase` should be placed in to `/tmp/wordpress-tests-lib/includes`.

https://github.com/miya0001/wp-cli/blob/master/templates/bootstrap.mustache#L15

And wp-tests-config.php should be placed parent directory of the `includes`.

```
$config_file_path = dirname( dirname( __FILE__ ) );
if ( ! file_exists( $config_file_path . '/wp-tests-config.php' ) ) { 
    // Support the config file from the root of the develop repository.
    if ( basename( $config_file_path ) === 'phpunit' && basename( dirname( $config_file_path ) ) === 'tests' )
        $config_file_path = dirname( dirname( $config_file_path ) );
}
$config_file_path .= '/wp-tests-config.php';
```
http://develop.svn.wordpress.org/trunk/tests/phpunit/includes/bootstrap.php.

Are there any problem?

Comment (miya0001) [inline templates/install-wp-tests.sh]: Ah, I forgot to test with `WP_TESTS_DIR` env ... 
I'll fix it.
